lib/docker/default.nix

```
@@ -2,13 +2,18 @@
 with lib; {
   copyDockerImages = { images, dest, args ? "" }:
     pkgs.writeScript "copy-docker-images.sh" (concatMapStrings
-      (image: ''
-        #!${pkgs.runtimeShell}
+      (image:
+        let
+          prefix = optionalString (image.isExe or false) "${image} | ${pkgs.gzip}/bin/gzip --fast |";
+          src = if image.isExe or false then "/dev/stdin" else ${image};
```

I think [the CI job](https://github.com/hall/kubenix/actions/runs/6770816713/job/18547902491?pr=39) doesn't like this line; maybe:

```suggestion
          src = if image.isExe or false then "/dev/stdin" else image;
```